openstates/in/utils.py

```diff
@@ -0,0 +1,22 @@
+import requests
+
+
+def get_with_increasing_timeout(scraper,link,fail=False,kwargs={}):
```

Comment:
OK, @mileswwatkins and I have conferred on this in person - we're trying to do something different here than scrapelib sessions deal with, since we're extending the time we wait for a timeout, not the time between attempts. So I think this line can be deemed resolved.